subsys/bluetooth/mesh/sensor_cli.c

```diff
@@ -954,11 +955,11 @@ int bt_mesh_sensor_cli_series_entry_get(
 
 	err = model_ackd_send(cli->model, ctx, &msg, rsp ? &cli->ack_ctx : NULL,
 			      BT_MESH_SENSOR_OP_COLUMN_STATUS, &rsp_data);
-	if (err) {
+	if (!rsp || err) {
 		return err;
 	}
 
-	if (rsp && rsp_data.count == 0) {
+	if (rsp_data.count == 0) {
 		return -ENOENT;
 	}
 
```

Comment:
Does this bit change anything at all, behavior-wise?

Reply:
This is just to follow the same pattern as in other functions.